mappings/net/minecraft/item/WrittenBookItem.mapping

```diff
@@ -1,5 +1,9 @@
 CLASS net/minecraft/class_1843 net/minecraft/item/WrittenBookItem
+	FIELD field_30929 MAX_TITLE_SELECTION_LENGTH I
 	FIELD field_30930 MAX_TITLE_LENGTH I
+	FIELD field_30931 MAX_PAGE_SELECTION_LENGTH I
+	FIELD field_30932 MAX_PAGES_LENGTH I
```

Comment:
Does this really have something to do with selection? It seems that the game just uses the different length limit for edit and view.

Reply:
You're right, changed